content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+* **Fix some memory leaks detected by miri**: Miri detected a case that leaks in our `BlobVec` drop impl. That is now fixed.
+* **Lint for missing SAFETY comments in bevy_ecs**: We now require safety comments for unsafe code blocks in `bevy_ecs`.
+
+As Bevy ECS matures, our bar for unsafe code blocks and soundness must also mature. Bevy ECS will probably never be 100% free of unsafe code blocks because we are modeling parallel data access that Rust cannot reason about without our help. But we are committed to removing as much unsafe code as we can and improving the quality and scope of our unsafe code.
+
+## Android Progress: We aren't there yet, but we're closer!
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+Bevy now "kind of" runs on Android again!
+
+However Android support _is not_ ready yet. There are issues with how we manage render contexts that must be resolved that sometimes break rendering at startup and _always_ break rendering when apps are minimized. Audio also doesn't work yet.
+
+![android](android.png)
+
+That being said, this is an important step forward, as Bevy developers can now build, deploy, (and in some cases test) Bevy apps on Android!
+
+If you are itching to test Bevy on mobile platforms, our [iOS support](https://github.com/bevyengine/bevy/blob/v0.8.0/examples/README.md#ios) is much more polished. Bevy developers have already started [publishing Bevy-based iOS apps to the Apple App Store](https://noumenal.app/).  
+
+## CI / Build System Improvements
+
+<div class="release-feature-authors">authors: @mockersf, @NiklasEi</div>
+
+As always, Bevy's CI had plenty of improvements this cycle:
+
+* Examples are now run in WASM when validating builds. Screenshots are taken and stored as part of the build outputs to ensure rendering works (`@mockersf`)
+* The Bevy examples are now run on a windows VM once per day to ensure they aren't broken (`@mockersf`)
+* License files are now automatically added to all published crates (`@NiklasEi`)
+* There is now a workflow to automatically generate a PR with version number bumps for all bevy crates (`@mockersf`)
+* To make the occasional nightly Rust breakage less disruptive, we've parameterized the nightly toolchain to make it easier to pin to a specific nightly. (`@mockersf`)
+
+## Example: Post Processing
+
+<div class="release-feature-authors">authors: @Vrixyz</div>
+
+Add 3d shapes example (#4613)
+Add a fun skinned mesh stress test based on the animated_fox example (#4674)
+add a post-processing example (#4797)
+Array texture example (#5077)
+    Use tone mapping in array_texture example (#5131)
+Add transparency examples (#3695)
+add a 3d lines example (#5319)
+
+## WASM Example Build Tool
+
+<div class="release-feature-authors">authors: @X</div>
+
+helper tool to build examples in wasm (#4776)
+
+## Website: Improved Examples Page
+
+<div class="release-feature-authors">authors: @doup</div>
```

Comment:
I think @ickk should also be mentioned here since they co-authored the original PR